Makefile

```diff
@@ -7,6 +7,9 @@ TAR ?= tar
 ZIP ?= zip
 PLANTUML ?= plantuml # may also be "java -jar plantuml.jar" if installed elsewhere
 
+KCONFIG_CONF ?= $(shell which kconfig-conf || which kbuild-conf || echo oldconfig)
```

Comment:
`which` is often missing on modern distros

Reply:
Probably better to explain the KCONFIG stuff is not required for installation with the default config.